onnxruntime/test/perftest/README.md

```diff
@@ -1,25 +1,63 @@
 # ONNXRuntime Performance Test
 
-onnxruntime_perf_test [options...] model_path result_file
+This tool provides the time for the ONNX Runtime with the specific execution provider to run a inference for a given model using the given test date. This tool can provide a reliable measurement for the inference latency usign ONNX Runtime on the HW platform. The options to use with the tool are listed below:
```

Comment:
test data

Reply:
instead of saying time may be performance data?

Reply:
__sample input__ test data?

Reply:
test data instead of test date

Reply:
updated